src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -3406,4 +3406,9 @@ public static void setConsecutiveMessageErrorsThreshold(int value)
     {
         conf.consecutive_message_errors_threshold = value;
     }
+
+    public static SubnetGroups getExcludeClientErrorsFrom()
```

Comment:
nit: Oddly, it feels like this would read better as...

```
if (DatabaseDescriptor.getClientErrorReportingExclusions().contains(ctx.channel().remoteAddress()))
```

i.e. "the exclusion set contains" in prose